variables.tf

```diff
@@ -233,3 +233,15 @@ variable "ip_address_timeout" {
   type        = string
   default     = "5m"
 }
+
+variable "nginx_controller" {
+  description = "Whether to have a NGINX Ingress Controller installed in this cluster; with a dedicated IP. Refer to the IP name in nginx_ip_names to be used here."
+  type = object({
+    enabled = bool
+    ip_name = string
+  })
+  default = {
+    enabled = false
+    ip_name = ""
```

Comment:
"" this is a value, if we don't send this variable nginx controller will try to create LB using empty ip which I guess will stuck there.
I would recommend to not specify a default value or use null.

Reply:
`enabled = false` so it will not even be activated.